LICENSES.txt

```diff
@@ -58,64 +58,64 @@ Apache-2.0
   gradle-tooling-api-7.3.jar
   gson-2.9.0.jar
   guava-27.0-jre.jar
-  guava-32.0.1-jre.jar
+  guava-31.0.1-jre.jar
   guice-4.0.jar
   guice-servlet-4.0.jar
-  hadoop-annotations-3.3.6.jar
-  hadoop-auth-3.3.6.jar
-  hadoop-common-3.3.6-tests.jar
-  hadoop-common-3.3.6.jar
-  hadoop-hdfs-3.3.6-tests.jar
-  hadoop-hdfs-3.3.6.jar
-  hadoop-hdfs-client-3.3.6.jar
-  hadoop-mapreduce-client-app-3.3.6.jar
-  hadoop-mapreduce-client-common-3.3.6.jar
-  hadoop-mapreduce-client-core-3.3.6.jar
-  hadoop-mapreduce-client-hs-3.3.6.jar
-  hadoop-mapreduce-client-jobclient-3.3.6-tests.jar
-  hadoop-mapreduce-client-jobclient-3.3.6.jar
-  hadoop-mapreduce-client-shuffle-3.3.6.jar
-  hadoop-minicluster-3.3.6.jar
-  hadoop-registry-3.3.6.jar
+  hadoop-annotations-3.3.5.jar
```

Comment:
Guava and Hadoop have all dropped, are you on the latest dev?

Reply:
Arrrrrgggg. At least the license step seems to be catching it now